### PR TITLE
HP-20 - Update gitops.json PREPROD

### PR DESCRIPTION
**Pull request by xritter1**
Link to Jira ticket if there is one:

### Environments
https://preprod.healdata.org/

### Description of changes
- new props: location, principle_investigator, publisher, source

### CODE REVIEW

File: preprod.healdata.org/portal/gitops.json [hunk 1]
```diff
@@ -133,7 +133,10 @@
         { "field": "category", "name": "Study Type"},
         { "field": "clinical_trial_website", "name": "Websites"},
         { "field": "nct_number", "name": "NCT Number"},
-        { "field": "publications", "name": "Study Publications"}
+        { "field": "publications", "name": "Study Publications"},
+        { "field": "priniciple_investigator", "name": "Principle Investigator"},
+        { "field": "publisher", "name": "Administering IC"},
+        { "field": "source", "name": "Research Program"}
```
Comment (ZakirG) [inline preprod.healdata.org/portal/gitops.json:139]: Should the "location" field be here as well?

Comment (xritter1) [reply preprod.healdata.org/portal/gitops.json:139]: yes!! thanks!

Comment (xritter1) [reply preprod.healdata.org/portal/gitops.json:139]: oh wait, I thought 'location" will be displayed in the portal also without the field mapping? location sits in rows 123 and 127. hm.. ?

Comment (mfshao) [reply preprod.healdata.org/portal/gitops.json:139]: you don't need to add it here if you are fine with `Location` as the field name displayed

**Review by ZakirG — APPROVED** [review]

**Pull request merged**